[2024-05-27T20:18:40.491+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-27T20:18:40.529+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_sales.get_send_data_task manual__2024-05-27T20:18:37.893777+00:00 [queued]>
[2024-05-27T20:18:40.543+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_sales.get_send_data_task manual__2024-05-27T20:18:37.893777+00:00 [queued]>
[2024-05-27T20:18:40.544+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2024-05-27T20:18:40.566+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_send_data_task> on 2024-05-27 20:18:37.893777+00:00
[2024-05-27T20:18:40.575+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4953) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-27T20:18:40.580+0000] {standard_task_runner.py:63} INFO - Started process 4955 to run task
[2024-05-27T20:18:40.579+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'process_sales', 'get_send_data_task', 'manual__2024-05-27T20:18:37.893777+00:00', '--job-id', '2452', '--raw', '--subdir', 'DAGS_FOLDER/process_sales.py', '--cfg-path', '/tmp/tmpdfh8h_5d']
[2024-05-27T20:18:40.583+0000] {standard_task_runner.py:91} INFO - Job 2452: Subtask get_send_data_task
[2024-05-27T20:18:40.653+0000] {task_command.py:426} INFO - Running <TaskInstance: process_sales.get_send_data_task manual__2024-05-27T20:18:37.893777+00:00 [running]> on host 6ecac41592ab
[2024-05-27T20:18:40.851+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='gurlenko.1488@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='process_sales' AIRFLOW_CTX_TASK_ID='get_send_data_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-27T20:18:37.893777+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-27T20:18:37.893777+00:00'
[2024-05-27T20:18:40.854+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-27T20:18:40.855+0000] {logging_mixin.py:188} INFO - Starting job 1
[2024-05-27T20:18:41.091+0000] {logging_mixin.py:188} INFO - Passion
[2024-05-27T20:18:41.094+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-27T20:18:41.095+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-27T20:18:41.133+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=process_sales, task_id=get_send_data_task, run_id=manual__2024-05-27T20:18:37.893777+00:00, execution_date=20240527T201837, start_date=20240527T201840, end_date=20240527T201841
[2024-05-27T20:18:41.217+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-27T20:18:41.255+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-27T20:18:41.258+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
